### PR TITLE
chore: add AGENTS.md, Makefile, and a ground-truth test fixture

### PR DESCRIPTION
**Pull request by BreakableHoodie**
## Summary
- `AGENTS.md` symlinks to `CLAUDE.md` so other AGENTS.md-aware tools (Codex, Cursor, etc.) get identical instructions with zero drift risk, per https://agents.md.
- `Makefile` wraps the existing npm scripts (`dev`, `build`, `check`, `lint`, `test`, etc.) for contributors who reach for `make` first.
- `tests/fixtures/ground-truth.json` centralizes synthetic geofence boundary points, merge-radius cases, and sample potholes instead of duplicating magic-number coordinates per spec file. Wired into `tests/e2e/geofence.spec.ts`, which also gained 3 previously-untested boundary cases (north/west/east) as a result — was 2 outside/1 inside case, now 5 outside/3 inside.

## Test plan
- [x] `npm run check` — 0 errors, 0 warnings
- [x] `npx tsc --noEmit` — clean
- [x] `npx playwright test tests/e2e/geofence.spec.ts` — 10/10 passed
- [x] Pre-commit hooks (eslint --fix, prettier --write) ran clean

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Tests**
  * Expanded geofence API coverage using deterministic fixture data with labeled, data-driven cases for inside/outside boundary points.
  * Added reusable ground-truth fixture data covering geofencing, report-merging expectations, and sample pothole records.
  * Kept validation checks for missing coordinates and geofence error handling.
* **Documentation**
  * Updated contributor guidance to point to the primary instructions document.
* **Chores**
  * Added a Makefile with streamlined development, build/preview, lint/type-check, and Playwright E2E (including accessibility) commands.

### CODE REVIEW

File: tests/e2e/geofence.spec.ts [hunk 1]
```diff
@@ -1,57 +1,52 @@
...
 
-	test('passes geofence for coordinates inside Waterloo Region', async ({ request }) => {
-		const response = await request.post('/api/report', {
-			data: { lat: 43.45, lng: -80.5 } // Central Kitchener
+			// Geofence passes — not a 422. With real DB creds this returns 200;
+			// with placeholder creds the downstream DB query fails (500), but the
+			// geofence logic itself is verified by the non-422 response.
+			expect(response.status()).not.toBe(422);
+			const body = await response.json();
+			expect(body.message ?? '').not.toMatch(/isn't in the Waterloo Region/i);
```
[the remark this answers is not in the document]
Comment (BreakableHoodie) [reply tests/e2e/geofence.spec.ts]: Fixed in 9186771 — narrowed the assertion to `expect([200, 500]).toContain(response.status())` so a 400 or 429 now fails the test instead of silently passing as a geofence success. Re-ran locally: 10/10 passed.

File: tests/fixtures/ground-truth.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      { "label": "Just west of western boundary", "lat": 43.45, "lng": -80.6 },
+      { "label": "Just east of eastern boundary", "lat": 43.45, "lng": -80.21 }
+    ]
+  },
+  "mergeRadius": {
+    "basePoint": { "lat": 43.4516, "lng": -80.4925 },
+    "duplicateReport": {
+      "lat": 43.45178,
+      "lng": -80.49224,
+      "note": "~18m from basePoint — should merge (within the 25m radius)"
```
[the remark this answers is not in the document]
Comment (BreakableHoodie) [reply tests/fixtures/ground-truth.json]: Confirmed via haversine — the original point was ~29m from basePoint, outside the 25m radius. Fixed in 9186771 using your suggested coordinates (verified ~17.8m).
